frontend/openchat-client/src/utils/media.ts

```diff
@@ -231,8 +221,8 @@ export async function messageContentFromFile(
             if (isImage) {
                 const extract = await extractImageThumbnail(blobUrl, mimeType);
 
-                if (data.byteLength > maxSizes.image) {
-                    data = (await resizeImage(blobUrl, mimeType, isDiamond)).data;
+                if (!isGif || data.byteLength > maxSizes.image) {
+                    data = (await stripMetaDataAndResize(blobUrl, mimeType)).data;
```

Comment:
But we should always do this, right? Not just when the byteLength exceeds the max.

Reply:
We do it always _if it's not a gif_ because I think it's less likely gifs will have meta data and I don't want to break _all_ gifs.

Reply:
Ahhh woops yup I misread that if statement